android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java

```diff
@@ -125,35 +125,42 @@ public void launchImageLibrary(final ReadableMap options, final Callback callbac
         boolean isPhoto = this.options.mediaType.equals(mediaTypePhoto);
         boolean isVideo = this.options.mediaType.equals(mediaTypeVideo);
 
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
+        boolean usePhotoPicker = Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU;
+
+        if (usePhotoPicker) {
```

Comment:
Order of the statements has been inversed to avoid a negation in the `if` statement